pysus/ftp/databases.py

```diff
@@ -0,0 +1,412 @@
+            year=year,
+            size=humanize.naturalsize(file.size),
+            last_update=file.date,
+        )
+
+        return description
+
+    def format(self, file: File) -> tuple:
+        if file.name == "DNEX2021":
+            pass
+
+        year = zfill_year(file.name[-2:])
+        charname = "".join([c for c in file.name if not c.isnumeric()])
+        uf = charname[-2:]
+        return uf, year
+
+    def get_files(
+        self,
+        ufs: Union[list[str], str],
+        years: Union[list, str, int],
+    ) -> List[File]:
+        if "EX" in to_list(ufs):
+            # DNEX2021
+            if len(to_list(ufs)) == 1:
+                return []
+            else:
+                ufs = to_list(ufs).remove("EX")
+
+        ufs = parse_UFs(ufs)
+        years = [str(y)[-2:].zfill(2) for y in to_list(years)]
+
+        # Fist filter years to reduce the list size
+        year_files = []
+        for file in self.files:
+            for year in years:
+                if year in file.name:
+                    year_files.append(file)
+
+        files = []
+        for file in year_files:
+            if "ANT/DNRES" in file.path:
+                for uf in ufs:
+                    if uf in file.name[3:]:
+                        files.append(file)
+            else:
+                for uf in ufs:
+                    if uf in file.name[2:]:
+                        files.append(file)
+
+        return files
+
+
+class SIH(Database):
+    name = "SINASC"
+    paths = [
+        "/dissemin/publicos/SIHSUS/199201_200712/Dados",
+        "/dissemin/publicos/SIHSUS/200801_/Dados",
+    ]
+    metadata = dict(
+        long_name="Sistema de Informações Hospitalares",
+        source=(
+            "https://datasus.saude.gov.br/acesso-a-informacao/morbidade-hospitalar-do-sus-sih-sus/",
+            "https://datasus.saude.gov.br/acesso-a-informacao/producao-hospitalar-sih-sus/",
+        ),
+        description=(
+            "A finalidade do AIH (Sistema SIHSUS) é a de transcrever todos os "
+            "atendimentos que provenientes de internações hospitalares que "
+            "foram financiadas pelo sus, e após o processamento, gerarem "
+            "relatórios para os gestores que lhes possibilitem fazer os pagamentos "
+            "dos estabelecimentos de saúde. Além disso, o nível Federal recebe "
+            "mensalmente uma base de dados de todas as internações autorizadas "
+            "(aprovadas ou não para pagamento) para que possam ser repassados às "
+            "Secretarias de Saúde os valores de Produção de Média e Alta complexidade "
+            "além dos valores de CNRAC, FAEC e de Hospitais Universitários – em suas "
+            "variadas formas de contrato de gestão."
+        ),
+    )
+    groups = dict(
+        RD="AIH Reduzida",
+        RJ="AIH Rejeitada",
+        ER="AIH Rejeitada com erro",
+        SP="Serviços Profissionais",
+        CH="Cadastro Hospitalar",
+        CM="",  # TODO
```

Comment:
@fccoelho do you know, by any chance, what `CM` means here? I couldn't find any reference to it. My guess is `Cadastro Municipal`, but I'm not sure

Reply:
No Idea, I have checked the documentation in the ftp but there is no mention of it. Leave it blank for now.